src/main/java/gov/nasa/jpl/mbee/mdk/mms/sync/delta/DeltaSyncRunner.java

```diff
@@ -322,10 +322,12 @@ else if (shouldUpdate && mmsChange != null) {
             if (!postElements.isEmpty()) {
                 try {
                     File file = MMSUtils.createEntityFile(this.getClass(), ContentType.APPLICATION_JSON, postElements, MMSUtils.JsonBlobType.ELEMENT_JSON);
+                    HashMap<String, String> uriBuilderParams = new HashMap<>();
+                    uriBuilderParams.put("overwrite", "true");
                     HttpRequestBase elementsUpdateCreateRequest = MMSUtils.prepareEndpointBuilderBasicJsonPostRequest(MMSElementsEndpoint.builder(), project, file)
                             .addParam(MMSEndpointBuilderConstants.URI_PROJECT_SUFFIX, projectId)
                             .addParam(MMSEndpointBuilderConstants.URI_REF_SUFFIX, refId)
-                            .addParam("overwrite", "true") // Prevent potential "Not Equivalent" errors due to default merging of element data.
+                            .addParam(MMSEndpointBuilderConstants.URI_BUILDER_PARAMETERS, uriBuilderParams)
```

Comment:
the delta sync doesn't check for contents key on server elements if the local change doesn't have a conflict, should leave off overwrite here and just keep it in manual commit

Reply:
removed